docs/getstdhandle.md

```diff
@@ -53,14 +53,21 @@ The standard device. This parameter can be one of the following values.
 
 | Value | Meaning |
 |-|-|
-| **STD_INPUT_HANDLE** (DWORD) -10 | The standard input device. Initially, this is the console input buffer, `CONIN$`. |
-| **STD_OUTPUT_HANDLE** (DWORD) -11 | The standard output device. Initially, this is the active console screen buffer, `CONOUT$`. |
-| **STD_ERROR_HANDLE** (DWORD) -12 | The standard error device. Initially, this is the active console screen buffer, `CONOUT$`. |
+| **STD_INPUT_HANDLE** `((DWORD)-10)` | The standard input device. Initially, this is the console input buffer, `CONIN$`. |
+| **STD_OUTPUT_HANDLE** `((DWORD)-11)` | The standard output device. Initially, this is the active console screen buffer, `CONOUT$`. |
+| **STD_ERROR_HANDLE** `((DWORD)-12)` | The standard error device. Initially, this is the active console screen buffer, `CONOUT$`. |
+
+> [!NOTE]
+> The values for these constants are unsigned numbers, but are defined in the header files as a cast from a 
+signed number and take advantage of the C compiler rolling them over to just under the maximum 32-bit value. When interfacing with these handles in a language that does not parse the headers and is re-defining the constants, please be aware of this constraint. As an example, `((DWORD)-10)` is actually the unsigned number `4294967286`.
 
 ## Return value
 
 If the function succeeds, the return value is a handle to the specified device, or a redirected handle set by a previous call to [**SetStdHandle**](setstdhandle.md). The handle has **GENERIC\_READ** and **GENERIC\_WRITE** access rights, unless the application has used **SetStdHandle** to set a standard handle with lesser access.
 
+> [!TIP]
+> It is not required to dispose of this handle with [**CloseHandle**](/windows/win32/api/handleapi/nf-handleapi-closehandle) when use is complete. See [**Remarks**](#handle-disposal) for more information.
```

Comment:
```suggestion
It is not required to dispose of this handle with [**CloseHandle**](/windows/win32/api/handleapi/nf-handleapi-closehandle) when you're done using it. See [**Remarks**](#handle-disposal) for more information.
```

Same below.

Reply:
I was told once upon a time to not involve "You" in technical documentation. I could change it to "when completed" or "when done" though.

Reply:
Oh yeah I remember having read the same. But I checked again and neither Microsoft's nor Google's technical writing guide mention anything against using "you". In fact they use it in their own examples.